crates/weaver_forge/templates/semconv_jq_fn/weaver.yaml

```diff
@@ -1,5 +1,5 @@
 params:
-  exclude_stability: ["experimental"]
+  exclude_stability: ["development"]
```

Comment:
This may also be a breaking change.  Should we update the default JQ expressions to treat "experimental" as `!= "stable"` ?

Reply:
it would be nice!

Reply:
Thanks for fixing this